### PR TITLE
Add un-released sticker features

### PR DESCRIPTION
**Pull request by DTS-11**

### CODE REVIEW

File: FEATURES.md [hunk 1]
```diff
@@ -63,6 +63,16 @@ These are divided in different categories depending on the object or concept the
...
+| Sticker Tags | Autocomplete/suggestion tags for stickers | Not Released | | | Documentation can be found [here](https://discord.com/developers/docs/resources/sticker) |
+| Sticker Type | Support for getting sticker types | Not Released | | | Documentation can be found [here](https://discord.com/developers/docs/resources/sticker#sticker-object-sticker-types) |
+| Sticker Available | Support for getting sticker availability | Not Released | | | Documentation can be found [here](https://discord.com/developers/docs/resources/sticker) |
+| Sticker Guild ID | Support for getting sticker guild ID | Not Released | | | Documentation can be found [here](https://discord.com/developers/docs/resources/sticker#sticker-object-sticker-guild-id) |
+| Sticker User | Support for getting the user who uploaded the sticker | Not Released | | | Documentation can be found [here](https://discord.com/developers/docs/resources/sticker) |
+| Sticker Sort Value | Standard sticker's sort value in its pack | Not Released | | | Documentation can be found [here](https://discord.com/developers/docs/resources/sticker) |
+
```
Comment (DA-344) [inline FEATURES.md:76]: Needs separator ``---``

File: FEATURES.md [hunk 2]
```diff
@@ -63,6 +63,16 @@ These are divided in different categories depending on the object or concept the
 
 ---
 
+### Stickers
```
Comment (DA-344) [inline FEATURES.md:67]: Needs to be added in the Contents Index

**Review by DA-344 — CHANGES_REQUESTED** [review]

**Comment by Soheab** [comment]
I've actually found all of these in discord.py..

tags: https://discordpy.readthedocs.io/en/stable/api.html#discord.StandardSticker.tags
type: ...
available: https://discordpy.readthedocs.io/en/stable/api.html#discord.GuildSticker.guild_id
guild id: https://discordpy.readthedocs.io/en/stable/api.html#discord.GuildSticker.guild_id
user: https://discordpy.readthedocs.io/en/stable/api.html#discord.GuildSticker.user
sort value: https://discordpy.readthedocs.io/en/stable/api.html#discord.StandardSticker.sort_value

Looks like only type isn't documented and I think that's because the class name tells you it anyways. But [it does set it](https://github.com/Rapptz/discord.py/blob/ff638d393d0f5a83639ccc087bec9bf588b59a22/discord/sticker.py#L340)

Unless i'm mistaken and these aren't the same thing?

**Comment by DTS-11** [comment]
bruh lmao, i searched using RoboDanny under Sticker class and couldnt find them ._.

**Comment by Soheab** [comment]
> bruh lmao, i searched using RoboDanny under Sticker class and couldnt find them ._.

 stickers are complicated.

**Comment by DTS-11** [comment]
yeah. I was doing more research and realized that app monetization are not included, should I make a PR for that?

**Comment by Soheab** [comment]
> yeah. I was doing more research and realized that app monetization are not included, should I make a PR for that?

Isn't that entitlements and skus?

**Comment by DTS-11** [comment]
but shouldnt there be a way to check if a user is a premium user?
ideally bot.is_premium_user idk

**Comment by Soheab** [comment]
> but shouldnt there be a way to check if a user is a premium user? ideally bot.is_premium_user idk

Yeah that would be nice but the API doesn't really provide such bool afaik and a check for it in the lib was denied in the following bikeshedding discussion: https://canary.discord.com/channels/336642139381301249/1256668426911617174

**Pull request closed without merging**